packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -22,10 +22,12 @@ interface IGroupedMessage {
 	compression?: string;
 }
 
-function isGroupContents(
-	opContents: IGroupedBatchMessageContents | { type?: unknown } | undefined,
-): opContents is IGroupedBatchMessageContents {
-	return opContents?.type === OpGroupingManager.groupedBatchOp;
+function isGroupContents(opContents: unknown): opContents is IGroupedBatchMessageContents {
+	return (
+		typeof opContents === "object" &&
```

Comment:
At runtime when is this `typeof` check actually necessary?  Why not call it `any` for this type guard so you can just do `?.` on it?  I don't know of an input where that will throw.

Reply:
Skipping the object check use of opContent: `Object is of type 'unknown'.ts(2571)` The javascript is perfectly legitimate as far as I know too.
I thought we didn't want to use `any`. I know in Whiteboard lint rules were enabled to catch its use. There are still uses for `any` and I am fine with its use here. It will be nice not to have code change.